packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsListCard.tsx

```diff
@@ -42,44 +41,45 @@ const StyledButton = styled.button`
   }
 `;
 
-type SettingsAccountsListCardProps<
-  Account extends Pick<ConnectedAccount, 'handle' | 'id'>,
-> = {
-  accounts: Account[];
+type Item = {
```

Comment:
Could be `SettingsAccountsListCardItem`? Wdyt?

Reply:
Renamed it!